config/development.yaml

```diff
@@ -80,8 +80,6 @@ clusterMetadata:
     active:
       enabled: true
       initialFailoverVersion: 0
-      rpcName: "cadence-frontend"
-      rpcAddress: "localhost:7933"
```

Comment:
shall we leave them here? People may not know we have the logic for filling default values for those fields and get confused.

Reply:
I feel like anything we have filled by default, should leave empty. 
This is already a convention in our config. 
Leaving them here will just make more work for users , and potentially, they will make it wrong. 
For example, a lot of users putting "localhost" here in the cluster setup, which would never work. 

So I still think we should remove them. But by addressing your another comment, I will remove the one in publicClient instead.